### DISCUSSION TITLE
Windows server system requirement

### DISCUSSION THREAD

**Discussion by fm2021fw** [kimai/kimai#2974] [On-Premise: Installation & Updates]
Hi,
I would like to install Kimai2 to a windows server, I cant't find system requirements into the kimai site. Is there anyone who knows if there are sytem requirements for windows server? For example if kimai2 works fine with windows server 2016 or anything like that?

Thanks regards
Fabio

**Comment by kevinpapst** [comment]
No idea if Windows Server 2016 works, but regular Kimai requirements are on the website: PHP >= 7.3 and Mysql or MariaDB
